ckanext/switzerland/templates/macros/form.html

```diff
@@ -35,5 +35,40 @@
     {% endif %}
     </div>
   {% endif %}
+{% endmacro %}
+
+{#
+Creates all the markup required for a date-type input element. Uses
+bootstrap-datepicker to allow localising the date.
+
+name        - The name of the form parameter.
+id          - The id to use on the input and label. Convention is to prefix with 'field-'.
+label       - The human readable label.
+value       - The value of the input.
+placeholder - Some placeholder text.
+error       - A list of error strings for the field or just true to highlight the field.
+classes     - An array of classes to apply to the form-group.
+is_required - Boolean of whether this input is requred for the form to validate
+
+Examples:
+
+{% import 'macros/form.html' as form %}
```

Comment:
Is this import needed? I might be mistaken, but doesn't this import the file itself?

Reply:
This line is inside the comment block for the macro, it's just showing an example of how to import it. :)

Reply:
oh my, didn't read this correctly then, sorry! 😅 thank you for clarifying 🙏